.travis.yml

```diff
@@ -1,5 +1,5 @@
 sudo: required
-dist: trusty
+dist: xenial
```

Comment:
This is not a valid value for the `dist` key. `trusty` is the most recent one.

Reply:
Hmm, why is this building then:

https://travis-ci.org/ros-drivers/odva_ethernetip/jobs/449282634

Reply:
Hmm, I see it is only experimental and sould not be used ...